### PR TITLE
[#54] Created `Team` component

### PR DESCRIPTION
**Pull request by Maciek246**

### CODE REVIEW

File: src/components/Dashboard/Form/DocumentForm.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { Component } from "react";
 import FormHeader from "./FormHeader/FormHeader.js";
 import { Form } from "react-final-form";
+import arrayMutators from 'final-form-arrays'
```
Comment (agatakozinska) [inline src/components/Dashboard/Form/DocumentForm.js]: no semicolon at the end of the line? :P

File: src/components/Dashboard/Form/Team/TeamMembers/TeamMembers.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    <FieldArray name="members">
+      {
+        ({ fields }) => (
+            <React.Fragment>
+              <button 
+                onClick={() => {fields.push({name: "", function: ""})}}>
+                  Dodaj
+              </button>
+              {
+                fields.map((element, index) =>
```
Comment (agatakozinska) [inline src/components/Dashboard/Form/Team/TeamMembers/TeamMembers.js]: for now it's ok to have index as a key, however docs says: 
>  Only do this if items have no stable IDs

and if we gonna store somewhere data about members (and I guess we will) so they will have unique ID that part should be changed in the future

Also @magul said that the first member will be always team-leader "function: "przewodniczący zespołu"" so we need add this exception

File: src/components/Dashboard/Form/DocumentForm.js (hunk 1 above)
Comment (Maciek246) [reply src/components/Dashboard/Form/DocumentForm.js]: Hmm in `Files changed` semicolon is on the right place. Probably comments to outdated commit or GitHub makes jokes.

File: src/components/Dashboard/Form/Team/TeamMembers/TeamMembers.js (hunk 2 above)
Comment (Maciek246) [reply src/components/Dashboard/Form/Team/TeamMembers/TeamMembers.js]: Yes, this is only a temporary solution until DB won't be connected. Every detail bounded with the future of this component will be planed later.

File: src/components/Dashboard/Form/DocumentForm.js (hunk 1 above)
Comment (agatakozinska) [reply src/components/Dashboard/Form/DocumentForm.js]: weird, I clicked on pending pull requests, my bad, then there was no comment here.

**Pull request merged**